### PR TITLE
Added support for Apache IvyDE editor

### PR DESCRIPTION
**Pull request by neskk**
It's my first ever git hub pull request!
Added a new mapping for Apache IvyDE editor.
Tested locally and seems to work as expected.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-1.1.0
+1.1.0 (2015-01-12)
```
Comment (fhd) [inline CHANGES.md]: You can leave this line as it is, we only add the date there when we actually make the release.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
 =====
+* Added support for Apache IvyDE editor (http://ant.apache.org/ivy/ivyde/)
```
Comment (fhd) [inline CHANGES.md]: How about using a Markdown link here, as in the README?

**Comment by fhd** [comment]
Thanks, looks good :smile:

**Comment by neskk** [comment]
Sorry for my newbie mistakes, shall I make the changes in CHANGED.md and commit them?
Thanks for your attention.

**Comment by fhd** [comment]
Yeah, you can just make a second commit changing this, I typically squash when merging anyway.

**Comment by neskk** [comment]
Will these changes be included in the official update site right away?

**Comment by fhd** [comment]
Sorry for the long delay, but it's merged now, thanks again! I'm planning to make a release when I'm back from vacation, i.e. in late July.

**Pull request closed without merging**